src/Features/Core/Portable/Workspace/ProjectCacheService.cs

```diff
@@ -48,12 +48,6 @@ public ProjectCacheService(Workspace? workspace, bool createImplicitCache = fals
             }
         }
 
-        /// <summary>
-        /// Recoverable trees only save significant memory for larger trees.
-        /// </summary>
-        public int MinimumLengthForRecoverableTree
-            => (_configurationService?.Options.DisableRecoverableTrees != true) ? 4 * 1024 : int.MaxValue;
```

Comment:
this option becamse always true.  which made this value always int.maxvaule, which allowed MinimumLengthForRecoverableTree to go away with this value being inlined at tis use site.